gsoc/proposal-Implementation of ML Framework to Prioritize Meaningful Changes In Environment Related Websites.md

```diff
@@ -0,0 +1,280 @@
+
+        3. Week 4 (June 20-June 29):
+
+            4. Test, debug and document code
+
+            5. Prepare work for Phase 1 submission along with a brief Phase 1 report
+
+            6. Prepare Pre-Phase 2 synopsis
+
+    7. Coding Period Phase 2 (June 30-July 27, 2017):
+
+        4. Week 5 & 6 (June 30-July 13):
+
+            7. Discuss the merits and demerits of different machine learning libraries and zero in on one.
+
+            8. Code a classifier for prioritization task (may be a logistic regression or a neural network)
+
+        5. Week 8 (July 21-July 28):
+
+            9. Begin implementing a framework on top of the classifier for the following feature (non-exhaustive):
+
+                1. Store and retrieve models
+
+            10. Prepare elaborate documentation and tests for Phase 2 components
+
+            11. Prepare work for Phase 2 submission
+
+            12. Prepare Pre-Phase 3 synopsis
+
+    8. Coding Period Phase 3 (July 29-August 29, 2017):
+
+        6. Week 9 & 10 (July 29-August 4):
+
+            13. Finish implementing the framework with the following features (non-exhaustive):
+
+                2. Retrain model by retrieving it and training it with new batch analyst feedback
+
+                3. Query test results for batch of diff data
+
+                4. Output the query result in specified format
+
+            14. Connect the framework to other EDGI services
+
+        7. Week 11 & 12 (August 5-August 25):
+
+            15. Test, debug code
+
+            16. Write pending documentation
+
+            17. Prepare Final Phase submission along with a detailed final phase report
+
+**Deliverables:**
+
+At the end of GSOC period I propose to deliver the following components:
+
+1. Code to convert diffs to a common format
+
+2. Code for Machine Learning classifier
+
+3. Framework code on top of the classifier to perform above mentioned functions
+
+4. Proper documentation for each component
+
+**Resources:**
+
+During my project, the following resources would play a crucial role and will facilitate my project work:
+
+1. People:
+
+    1. Environmental Data Experts and Analyst, for their opinion to zero in on the best possible criteria for meaningful changes in a website
+
+    2. Webpage designers for their opinion on which kind of impact a change in a web page can have on a user
+
+    3. People with experience with web monitoring services to help in quick understanding of diff generating APIs
+
+2. Documentation:
+
+    4. Documentation of web monitoring services being used
+
+    5. Documentation of other EDGI components for coding and integration process
+
+3. Literature:
+
+    6. Machine learning literature for designing the classifier
+
+4. Sample Data:
+
+    7. Training data of web page changes (diffs) for training
+
+**Setup:**
+
+* Do develop familiarity with EDGI codebase I have setup [web-monitoring-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) locally. 
```

Comment:
If you have time, you might do a quick read-through for misspellings and grammar. Overall, the proposal is in great shape, but there are a few minor things throughout. E.g. the above should be:

"To develop familiarity with the EDGI codebase I have set up..."